src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -70,7 +72,8 @@ public class FlowManager : IFlowManager
             Application application,
             IFlowLoader flowLoader,
             IFlowSessionManager flowSessionManager,
-            IAnalyzeBuilderExceptions analyzeBuilderExceptions
+            IAnalyzeBuilderExceptions analyzeBuilderExceptions,
+            IInputMessageHandler inputReplyHandler
```

Comment:
Suggestion: FlowManager is already a huge class, we should avoid to add more dependencies and group related dependencies on another abstraction.

Based on this idea, we could for instance make `IInputMessageHandler` have all the methods from `IInputExpirationHandler`, and have an `AggregateInputMessageHandler` that will iterate over all `IInputMessageHandler`?

Something like this:
```csharp
public class AggregateInputMessageHandler : IInputMessageHandler
{
   public AggregateInputMessageHandler(IEnumerable<IInputMessageHandler> handlers) 
  {  
     // save handlers to private field 
  }

   // implement each method from  IInputMessageHandler by iterating over received handlers from constructor 
} 

// In ContainerExtensions, something like (I don't really know if this works as is, but some variation should do it):
  container. RegisterSingleton<IInputMessageHandler,  AggregateInputMessageHandler>();
             container.Collection.Register<IInputExpirationHandler>(
                new[]
                {
                    typeof(InputExpirationHandler)
                    typeof(InputReplyHandler)
                },
                Lifestyle.Singleton);
```

Reply:
The InputExpirationHandler implements IInputExpirationHandler too.
This implementation won't avoid to add more dependencies unless we take the IInputExpirationHandler instance dinamically in the collection on FlowManager...
Howover, thinking in the future, this implementation brings more organization in the source code when we needs to add a new IInputMessageHandler.

The correct method to call in this context is "container.RegisterMany"